LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2014 Tony Morris, Runar Bjarnason, Tom Adams, Kristian Domagala, Brad Clow, Ricky Clarkson, Paul Chiusano, Trygve Laugstøl, Nick Partridge, Jason Zaugg
+Copyright (c) 2009-2018 Tony Morris, Runar Bjarnason, Tom Adams, Kristian Domagala, Brad Clow, Ricky Clarkson, Paul Chiusano, Trygve Laugstøl, Nick Partridge, Jason Zaugg
```

Comment:
I really think we should add @xuwei-k to this list. Is there a reason you have not added yourself?

Reply:
Honestly I'm confused by this whole names-in-licensing situation. Obviously the copyright is held by more people than this, and this list doesn't match what's in the `pomExtra`. 

Would it be the best to just say "the ScalaZ contributors"? That's not false. I could go through the git history and make a `CONTRIBUTORS` file that we could pinky-swear to keep up-to-date.

Also, is there a reason that _you_ haven't added _yourself?

Reply:
The super-high-tech solution is to keep a list of people lying around, then have the package step prepend `Copyright © 2009-$currentYear ${folks mkString ", "}` to this file. Then we could add the same to the `pomExtra`.

Reply:
Thing with BSD-3 is you can't change it. Technically you'd need everybody to agree to even change this, but I don't think anybody will mind a year bump and extra names from real contributors.

I don't feel I've added enough to 7.2 to justify adding my name. But I've added it in 7.3 because I added the `Deriving` base typeclasses and it took me a lot of time to figure that out.

Yeah, it should be everybody's name. Apache 2.0 encourages a `NOTICE` file, which can even be generated from the git history.

Reply:
Is this line technically part of the license, then? That seems strange, given that under BSD everyone retains their copyright whether or not they're here. Would "Copyright 2009-2018, the Contributors (see included CONTRIBUTORS file)" pass muster, then?

Reply:
yes this line is part of the license, and no it's not conventional for a BSD license to refer to a contributors file because it isn't included in the license. You'd have the words "copyright the contributors, see in the included file" with no obligation for anybody to actually share that file. So you'd end up creating a 4th clause. etc etc etc.

Reply:
@hrhino I like that solution for scalaz8. We'd need to have a version of the file itself for our own source distribution.

Reply:
Right, so, I was thinking "copyright the contributors\n${bsd license}" in `LICENSE`, and when it's packaged we insert the names into that and the pom.

But that presumably wouldn't be legit, because it would imply that we're distributing it under a (textually) different LICENSE than what people see looking at the repo, and a different one than the previous version whenever a new contributor joins the cabal.

> ... with no obligation for anybody to actually share that file. So you'd end up creating a 4th clause.

BSD4, you saw it here first